Tasks/DownloadBuildArtifacts/task.loc.json

```diff
@@ -8,7 +8,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 132,
+    "Minor": 134,
```

Comment:
These files where automatically updated. I'm not sure if this an expected behavior